src/com/ableton/VirtualEnv.groovy

```diff
@@ -7,6 +7,7 @@ package com.ableton
  */
 class VirtualEnv implements Serializable {
   def script
+  def python
```

Comment:
sort above `def script`?

Reply:
I would prefer that the order of declaration matches the order in which they are required in the constructor.